plugins/baseplugin/src/main/java/org/apache/unomi/plugins/baseplugin/actions/CopyPropertiesAction.java

```diff
@@ -112,4 +113,9 @@ public int execute(Action action, Event event) {
 
         return propsToCopy;
     }
+
+    private String getPropertyName(String propertyName) {
```

Comment:
I wonder if we should rename this to resolvePropertyName, getPropertyName seems weird to me.